app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx

```diff
@@ -249,6 +249,11 @@ export function NoticeTypeCheckboxes({
     JsonNoticeTypeLinks.Chime = '/missions/chime'
   }
 
+  if (useFeature('KM3NET_QUICKSTART')) {
```

Comment:
Could we call the feature flag:

```suggestion
  if (useFeature('KM3NET')) {
```